presto-common/src/test/java/com/facebook/presto/common/zorder/TestZOrder.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.common.zorder;
+
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class TestZOrder
+{
+    private void printBits(byte[] bytes)
+    {
+        int byteCount = 0;
+        for (byte b : bytes) {
+            System.out.print(Integer.toBinaryString(Byte.toUnsignedInt(b)));
+            byteCount++;
+        }
+        System.out.println();
+        System.out.println(byteCount);
+    }
+
+    private static int getHighestSetBitPosition(long value)
```

Comment:
Why the paramter is long not integer?

Reply:
I made the parameter a long because in the past I needed to use the function on a long. int and byte types automatically cast up into long as well, so this does not affect them. Right now, there is no problem turning the type into an int, so I can do that.